src/cursor/cur_backup.c

```diff
@@ -220,6 +272,9 @@ __curbackup_close(WT_CURSOR *cursor)
             WT_WITH_TXN_ISOLATION(
               session, WT_ISO_READ_COMMITTED, ret = __wt_checkpoint(session, cfg))));
     }
+    /* Clear the flag on force stop after the completion of the checkpoint. */
```

Comment:
Most of the cherry-pick was clean but I want to call out this section for extra scrutiny since I needed to resolve a merge conflict. 
On 7.2 and later we also have the changes from WT-11297. I don't think they're necessary and we only needed to add the clearing of the flag.

Reply:
Thanks for calling out this area.